.circleci/prepare.sh

```diff
@@ -0,0 +1,8 @@
+$PYTHON --version
```

Comment:
As to the shell script, I've since learned how to reuse stuff both in yaml and specifically within circle config.  I'll plan to do that and drop the `.sh`.

Reply:
Looking forward to that :)